unison-cli/src/Unison/Codebase/Editor/HandleInput.hs

```diff
@@ -2438,43 +2440,57 @@ mergeBranchAndPropagateDefaultPatch ::
   LoopState.InputDescription ->
   Maybe (Output v) ->
   Branch m ->
-  Maybe Path.Path' ->
```

Comment:
We were indicating verbosity by whether this one single path was a Just or Nothing, which seemed very strange to me, so now we just pass verbosity explicitly.

We should definitely investigate a more robust 'logging' setup in the future.